src/main/pages/che-7/administration-guide/proc_deploying-kubernetes-image-puller-using-openshift-templates.adoc

```diff
@@ -36,40 +36,20 @@ See xref:kubernetes-image-puller-configuration[] for more information about conf
 
 *Installing*
 
-. Clone the `kubernetes-image-puller` repository:
+. Clone and go to the `kubernetes-image-puller` repository:
 +
 ----
 $ git clone https://github.com/che-incubator/kubernetes-image-puller
 $ cd kubernetes-image-puller
 ----
 
-. Log into the OpenShift cluster and create a namespace to deploy the puller into:
+. Prepare OpenShift to receive the deployment of the puller:
```

Comment:
Not sure if this wording is right e.g. `to receive the deployment of `

Reply:
I agree

Reply:
It was an attempt to make it context agnostic. 

Upstream we will have:

```
oc new-project k8s-image-puller
```

I believe both variants are equivalent.

Or we could use the same command for upstream and downstream.